### PR TITLE
snippets - show visible elec

### PR DESCRIPTION
**Pull request by magland**

### CODE REVIEW

File: src/extensions/snippets/SnippetsView/SnippetBox.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { FunctionComponent, useCallback, useState } from 'react';
+import React, { FunctionComponent, useCallback, useMemo, useState } from 'react';
```
Comment (magland) [inline src/extensions/snippets/SnippetsView/SnippetBox.tsx:1]: The changes in this file somehow did not get merged into master - they synchronize the current timepoint with the snippets view

File: src/extensions/snippets/SnippetsView/SnippetsRow.tsx [hunk 2]
```diff
@@ -109,9 +109,9 @@ const createTimeSegments = (timeRange: {min: number, max: number} | null, opts:
     return ret
 }
 
-const useSnippets = (args: {recording: Recording, sorting: Sorting, unitId: number, timeRange: {min: number, max: number} | null}) => {
+const useSnippets = (args: {recording: Recording, sorting: Sorting, visibleElectrodeIds: number[] | undefined, unitId: number, timeRange: {min: number, max: number} | null}) => {
```
Comment (magland) [inline src/extensions/snippets/SnippetsView/SnippetsRow.tsx:112]: Restrict snippets to only the visible electrodes

**Review by jsoules — COMMENTED** [review]
Seems fine, I made one suggestion to tweak some line breaks/spacing to make some structure more clear but feel free to ignore that.

File: src/extensions/snippets/SnippetsView/SnippetsRow.tsx [hunk 3]
```diff
@@ -140,19 +140,32 @@ const useSnippets = (args: {recording: Recording, sorting: Sorting, unitId: numb
             snippetsList.reduce((acc, val) => (
                 val !== undefined ? acc?.concat(val) : acc
             ), [] as Snippet[])
-        ) : undefined
+        )?.map(s => (filterSnippetVisibleElectrodes(s, info?.channel_ids, visibleElectrodeIds))) : undefined
```
[the remark this answers is not in the document]
Comment (magland) [reply src/extensions/snippets/SnippetsView/SnippetsRow.tsx:143]: Submitted an issue for this.

**Pull request merged**